src/main.rs

```diff
@@ -464,7 +463,12 @@ impl App {
                 input
             }
             Some(fname) => std::fs::read_to_string(fname)?,
-            None if args.get_flag("from-reference") => stub::SIMPLE_REFERENCE_STUB.to_owned(),
+            None if args.get_flag("from-reference") => {
+                let reference_clash = __test_helper::sample_puzzle("stub_and_solution_tester").unwrap();
+                let reference_stub = reference_clash.stub_generator().unwrap();
+
+                reference_stub.to_owned()
+            }
```

Comment:
I don't like that this turns a simple constant into something that does file IO and JSON parsing. And I think it would crash when running `coctus` from outside the repository as the file won't be there(?).

Did you consider using [`include_str!`](https://doc.rust-lang.org/std/macro.include_str.html) to get the reference stubgen input directly from a file at compile time? That would be short enough to do directly in `main.rs` without being distracting.

Reply:
The strong selling point for me was how easy it would be to sync the test with a CG contribution.

Where should the stub generator be located if I were to use `include_str!`?